mapreduce/slowsql/filter_template.json

```diff
@@ -2,28 +2,40 @@
   "version": 1,
   "dimensions": [
     {
-      "field_name": "reason",
-      "allowed_values": ["saved-session"]
+      "field_name": "prefix",
+      "allowed_values": "telemetry-2"
     },
     {
-      "field_name": "appName",
-      "allowed_values": "*"
+      "field_name": "submissionDate",
+      "allowed_values": ["__TARGET_DATE__"]
+    },
+    {
+      "field_name": "sourceName",
+      "allowed_values": "telemetry"
+    },
+    {
+      "field_name": "sourceVersion",
+      "allowed_values": "4"
+    },
+    {
+      "field_name": "docType",
+      "allowed_values": "saved_session"
     },
     {
-      "field_name": "appUpdateChannel",
+      "field_name": "appName",
       "allowed_values": "*"
     },
     {
-      "field_name": "appVersion",
+      "field_name": "channel",
```

Comment:
`s/channel/appUpdateChannel/` for consistency with the field name inside 'meta/'